mcs/class/System.Transactions/System.Transactions/Transaction.cs

```diff
@@ -69,10 +69,10 @@ right now
 
 		internal IPromotableSinglePhaseNotification Pspe { get { return pspe; } }
 
-		internal Transaction ()
+		internal Transaction (TransactionOptions options)
```

Comment:
I think it'd be better to only pass IsolationLevel

Reply:
Yes, it's better this way